CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [1.0.0-dev.9]
+
+- fix: update supabase to [v1.0.0-dev.8](https://github.com/supabase-community/supabase-dart/pull/146)
```

Comment:
Is there a reason of linking to the pr here, but linking to the changelog at the realtime update in line 7?

Reply:
I haven't been consistent with this, but I am trying to write the changelog in the following format:

- supabase_flutter update
- update supabase version
  - contents of the supabase version update
  - update sub-library version 
    - contents of the sub-library version update

Reply:
What do you think about this format? Too complicated?

Reply:
What do you think about this format? Too complicated?

Reply:
I like that you don't have to read every's package changelog individually, but you link to the pr on supabase update, but to the changelog on realtimee update.

Reply:
Ah, I see. Nice catch!